### PR TITLE
feat: team activity

### PR DESCRIPTION
**Pull request by bashbunni**
based on the latest pushes to repos in an org

### CODE REVIEW

**Review by ccoVeille — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -180,6 +180,23 @@ Published: {{humanize .LastRelease.PublishedAt}}
 This function requires GitHub authentication with the following API scopes:
 `repo:status`, `public_repo`, `read:user`.
 
+### Recent pushes in an organization
```
Comment (bashbunni) [inline README.md]:
```suggestion
### Recent pushes
```

File: repos.go [hunk 2]
```diff
@@ -282,6 +282,59 @@ func recentReleases(count int) []Repo {
...
+		"owner": githubv4.String(owner),
+	}
+	err := gitHubClient.Query(context.Background(), &query, variables)
+	if err != nil {
+		panic(err)
+	}
+
+	for _, v := range query.Owner.Repositories.Edges {
+		// ignore meta-repo
+		repos = append(repos, repoFromQL(v.Node))
```
Comment (aymanbagabas) [inline repos.go:372]: You're _not_ ignoring anything here 😄 

```suggestion
		if string(v.Node.NameWithOwner) == fmt.Sprintf("%s/%s", username, username) {
			continue
		}
		
		repos = append(repos, repoFromQL(v.Node))
```

Comment (bashbunni) [reply repos.go:372]: Oh lol! I forgot to delete the comment. I think it's okay if we include the meta repo in our recent activity 🤔 What do you think?

**Comment by bashbunni** [comment]
- [x] do recentCreatedRepos and recentPushedRepos -> rename existing queries in this PR to match this

**Review by aymanbagabas — APPROVED** [review]

**Pull request merged**